backends/tfhe-cuda-backend/cuda/src/integer/comparison.cuh

```diff
@@ -135,7 +135,7 @@ __host__ void are_all_comparisons_block_true(
             is_max_value_lut->get_degree(1),
             is_max_value_lut->get_max_degree(1), glwe_dimension,
             polynomial_size, message_modulus, carry_modulus,
-            is_equal_to_num_blocks_lut_f);
+            is_equal_to_num_blocks_lut_f, true);
```

Comment:
Have you considered having two versions of `generate_device_accumulator` as you did in the device functions?>

Reply:
No, I didn't think it was too bad to add the extra argument for those.